test_scripts/Smoke/API/042_SubscribeButton_Non_Media_PositiveCase_SUCCESS.lua

```diff
@@ -44,29 +52,26 @@ local buttonName = {
   "PRESET_5",
   "PRESET_6",
   "PRESET_7",
-  "PRESET_8"
-}
-
-local mediaButtonName = {
-  "PLAY_PAUSE",
-  "SEEKLEFT",
-  "SEEKRIGHT",
-  "TUNEUP",
-  "TUNEDOWN"
+  "PRESET_8",
+  "PRESET_9"
 }
 
 --[[ Local Functions ]]
 local function subscribeButton(pButName, self)
   local cid = self.mobileSession1:SendRPC("SubscribeButton", { buttonName = pButName })
   local appIDvalue = commonSmoke.getHMIAppId()
-  EXPECT_HMINOTIFICATION("Buttons.OnButtonSubscription", { appID = appIDvalue, name = pButName, isSubscribed = true })
+  EXPECT_HMICALL("Buttons.SubscribeButton",{ appID = appIDvalue, buttonName = pButName })
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
+  EXPECT_HMINOTIFICATION("Buttons.OnButtonSubscription")
```

Comment:
@GetmanetsIrina According to the proposal `OnButtonSubscription` has to be removed

Reply:
@dboltovskyi , updated.